themes/sbvfrd/templates/subjectdetailpage/relations/authors.phtml

```diff
@@ -0,0 +1,39 @@
+<? if ($this->subjectAuthorsTotal > 0): ?>
+<div class="carousel-container">
+    <a id="detailpage-coauthors"></a>
+    <div class="separator"></div>
+    <div class="row">
+        <div class="col-xs-8">
+            <h2><?= $this->essubject()->getSearchResultHeadline('page.detail.authors.subjects', $this->subjectAuthorsTotal, true) ?></h2>
+        </div>
+        <div class="col-xs-4">
+            <span class="all-results-link">
+                <a href="<?= $this->esperson()->getCoauthorsSearchLink() ?>"><?= $this->translate('common.show.results.all') ?></a>
+            </span>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-xs-12">
+            <?= $this->render(
+                '/detailpage/carousel.phtml', [
+                    'id' => 'coauthors',
+                    'pagination' => ['xs' => 2, 'sm' => 2, 'md' => 4, 'lg' => 6],
+                    'thumbnail' => $this->imageLink('person-placeholder.png'),
+                    'total' => $this->subjectAuthorsTotal,
+                    'templates' => [
+                        'ajax' => $this->ajax([
```

Comment:
@outermedia-cbo below you'll see how I named the method that is passed through the Ajax query parameters.